sys/dev/sound/pcm/channel.c

```diff
@@ -1236,6 +1236,8 @@ chn_init(struct snddev_info *d, struct pcm_channel *parent, kobj_class_t cls,
 	}
 
 	PCM_UNLOCK(d);
+	b = NULL;
+	bs = NULL;
```

Comment:
Is there a reason we don't just initialize these to null where they are declared above?